src/patternfly/components/DataList/data-list-expandable-content-body.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="pf-c-data-list__expandable-content-body{{#if data-list-expandable-content-body--modifier}} {{data-list-expandable-content-body--modifier}}{{/if}}"
```

Comment:
Same comment here. I don't think we want to limit the elements they can use in this element.